public/app/features/alerting/unified/hooks/useAlertQueriesStatus.ts

```diff
@@ -1,11 +1,18 @@
 import { useMemo } from 'react';
 
 import { getDataSourceSrv } from '@grafana/runtime';
+import { isExpressionReference } from '@grafana/runtime/src/utils/DataSourceWithBackend';
```

Comment:
The import path is a bit suspicious, however, other files import `isExpressionReference` in the same way

Reply:
Yea I've been trying to get around it but figured it was fine since it's indeed imported as such in other places.